### ISSUE TITLE
Responsive Font Size Accessibility

### ISSUE THREAD

**Issue by jolanglinais** [accordproject/template-studio-v2#43]
# WCAG 2.0 Level AA [1.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html) (Resize Text)

## Tasks
- All fonts should be sized relatively, which will be more responsive and allow better accessibility to those who magnify or zoom on their devices. Move away from `px` and instead favor `rem`, `em`, or `%`.

## Related Issues: 
- Issue [#58](https://github.com/accordproject/cicero-ui/issues/58) in Cicero-UI

**Comment by florenciasilva** [comment]
I would love to work on this!

**Comment by jolanglinais** [comment]
Great @florenciasilva! 🤗

Let us know if you need any help. We'll be counting any PRs for this in October for Hacktoberfest.

**Comment by florenciasilva** [comment]
Hello, @irmerk ! 
I changed all the `font-size` I could find in this repository, but I do notice that some may be coming from cicero-ui. 
Should I send this PR and move to changing the remaining `font-size`'s of cicero-ui?

**Comment by jolanglinais** [comment]
Great! Looking at the PR now. Yeah, I think the related work in the issue you referenced in your PR is good to continue this thread ([Cicero UI #58](https://github.com/accordproject/cicero-ui/issues/58))

**Comment by jolanglinais** [comment]
Thanks @florenciasilva! Adding you to the little spreadsheet I'm keeping for AP Hacktoberfest contributions.

I'll close this issue but anyone feel free to reopen if there is more work here in the future.
